### PR TITLE
Added support for unbounded int fields

### PR DESCRIPTION
**Pull request by yuce**

### CODE REVIEW

File: pilosa/orm.py [hunk 1]
```diff
@@ -475,7 +475,7 @@ class Field:
     def __init__(self, index, name, time_quantum,
                  cache_type, cache_size, int_min, int_max, keys, mutex, bool):
         validate_field_name(name)
-        if int_max < int_min:
+        if int_min is not None and int_max is not None and int_max < int_min:
```
Comment (alanbernstein) [inline pilosa/orm.py]: I don't have strong feelings about this, but should the client pass this through, and behave the same way as go-pilosa?

Comment (yuce) [reply pilosa/orm.py]: ok, updated.

**Comment by yuce** [comment]
I have no idea about the `celery` stuff. I've ran the tests on a fresh virtualenv but there are no problems on my side running tests. I am very curious about why running tests required celery on your side and how you decided to install it though :)

**Comment by alanbernstein** [comment]
This is really all I know: 
```
$ make test-all
py.test tests integration_tests
Traceback (most recent call last):
  File "/usr/local/bin/py.test", line 10, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.7/site-packages/_pytest/config.py", line 49, in main
    config = _prepareconfig(args, plugins)
  File "/usr/local/lib/python3.7/site-packages/_pytest/config.py", line 168, in _prepareconfig
    pluginmanager=pluginmanager, args=args)
  File "/usr/local/lib/python3.7/site-packages/_pytest/vendored_packages/pluggy.py", line 745, in __call__
    return self._hookexec(self, self._nonwrappers + self._wrappers, kwargs)
  File "/usr/local/lib/python3.7/site-packages/_pytest/vendored_packages/pluggy.py", line 339, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File "/usr/local/lib/python3.7/site-packages/_pytest/vendored_packages/pluggy.py", line 334, in <lambda>
    _MultiCall(methods, kwargs, hook.spec_opts).execute()
  File "/usr/local/lib/python3.7/site-packages/_pytest/vendored_packages/pluggy.py", line 613, in execute
    return _wrapped_call(hook_impl.function(*args), self.execute)
  File "/usr/local/lib/python3.7/site-packages/_pytest/vendored_packages/pluggy.py", line 250, in _wrapped_call
    wrap_controller.send(call_outcome)
  File "/usr/local/lib/python3.7/site-packages/_pytest/helpconfig.py", line 68, in pytest_cmdline_parse
    config = outcome.get_result()
  File "/usr/local/lib/python3.7/site-packages/_pytest/vendored_packages/pluggy.py", line 279, in get_result
    raise ex[1].with_traceback(ex[2])
  File "/usr/local/lib/python3.7/site-packages/_pytest/vendored_packages/pluggy.py", line 265, in __init__
    self.result = func()
  File "/usr/local/lib/python3.7/site-packages/_pytest/vendored_packages/pluggy.py", line 614, in execute
    res = hook_impl.function(*args)
  File "/usr/local/lib/python3.7/site-packages/_pytest/config.py", line 957, in pytest_cmdline_parse
    self.parse(args)
  File "/usr/local/lib/python3.7/site-packages/_pytest/config.py", line 1121, in parse
    self._preparse(args, addopts=addopts)
  File "/usr/local/lib/python3.7/site-packages/_pytest/config.py", line 1084, in _preparse
    self.pluginmanager.load_setuptools_entrypoints('pytest11')
  File "/usr/local/lib/python3.7/site-packages/_pytest/vendored_packages/pluggy.py", line 510, in load_setuptools_entrypoints
    plugin = ep.load()
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2411, in load
    return self.resolve()
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2417, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 668, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 638, in _load_backward_compatible
  File "/usr/local/lib/python3.7/site-packages/_pytest/assertion/rewrite.py", line 212, in load_module
    py.builtin.exec_(co, mod.__dict__)
  File "/usr/local/lib/python3.7/site-packages/celery/contrib/pytest.py", line 6, in <module>
    from .testing import worker
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 668, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 638, in _load_backward_compatible
  File "/usr/local/lib/python3.7/site-packages/_pytest/assertion/rewrite.py", line 212, in load_module
    py.builtin.exec_(co, mod.__dict__)
  File "/usr/local/lib/python3.7/site-packages/celery/contrib/testing/worker.py", line 6, in <module>
    from celery import worker
  File "/usr/local/lib/python3.7/site-packages/celery/worker/__init__.py", line 3, in <module>
    from .worker import WorkController
  File "/usr/local/lib/python3.7/site-packages/celery/worker/worker.py", line 241
    self._conninfo.transport.implements.async and
                                            ^
SyntaxError: invalid syntax
make: *** [test-all] Error 1
```

**Comment by yuce** [comment]
Most probably an older version of celery is installed in your system with a conflicting dependency. The issue would be resolved If you uninstall it or use a fresh virtualenv.

**Pull request merged**